pyproject.toml

```diff
@@ -25,6 +25,9 @@ flake8 = "^6.0.0"
 ipython = "^8.8.0"
 pydocstyle = "^6.3.0"
 testcontainers = "^3.7.1"
+mypy = "^0.991"
+pandas-stubs = "^1.5.2.230105"
```

Comment:
these seem odd?

Reply:
What do you mean?

Reply:
https://pypi.org/project/mypy/
https://pypi.org/project/pandas-stubs/